src/Bridge/Symfony/Bundle/Test/ApiTestCase.php

```diff
@@ -69,4 +69,17 @@ protected static function createClient(array $kernelOptions = [], array $default
 
         return $client;
     }
+
+    /**
+     * Finds the IRI of the resource of the given class, corresponding to the given criteria.
+     */
+    protected function findIriBy(string $resourceClass, array $criteria): ?string
+    {
+        $resource = static::$container->get('doctrine')->getRepository($resourceClass)->findOneBy($criteria);
```

Comment:
Would fail if resource class is not a Doctrine class.

Reply:
Fixed in last commit, thanks!